tests/component-tests/pager/js/PagedTableTests.js

```diff
@@ -150,60 +159,113 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
     };
 
     fluid.tests.tooltipModuleSource = function (pager) {
-        var pageLinksTop = $("a", pager.pagerBar.locate("pageLinks"));
-        var pageLinksBottom = $("a", pager["pagerBar-1"].locate("pageLinks"));
-
-        var tooltipContents = [
-            [
-                {nodeName: "b", nodeText: "bird"},
-                {nodeName: "b", nodeText: "fish"}
-            ]
-        ];
-
         var sequence = [];
 
-        var assertVisibleTips = function (message, targetIds) {
-            sequence.push({
-                event: "{trackTooltips}.events.notifyFocusChange",
-                listener: function () {
-                    fluid.tests.tooltip.assertVisible(message, pager, targetIds, null, function (tooltip) {
-                        jqUnit.assertNode("The contents of the tooltip should be set", tooltipContents[0], $("b", tooltip));
-                    });
+        /* Change the page size and verify whether the tooltips are synced accordingly */
+
+        fluid.tests.setPageSize(pager, sequence, 2);
+        fluid.tests.verifyToolTips(pager, sequence, 2, [
+            [ {nodeName: "b", nodeText: "dog"}, {nodeName: "b", nodeText: "cat"} ],
+            [ {nodeName: "b", nodeText: "bird"}, {nodeName: "b", nodeText: "fish"} ],
+            [ {nodeName: "b", nodeText: "camel"}, {nodeName: "b", nodeText: "dragon"} ],
+            [ {nodeName: "b", nodeText: "ant"}, {nodeName: "b", nodeText: "ant"} ]
+        ]);
+
+        fluid.tests.setPageSize(pager, sequence, 3);
+        fluid.tests.verifyToolTips(pager, sequence, 3, [
+            [ {nodeName: "b", nodeText: "dog"}, {nodeName: "b", nodeText: "bird"} ],
+            [ {nodeName: "b", nodeText: "fish"}, {nodeName: "b", nodeText: "dragon"} ],
+            [ {nodeName: "b", nodeText: "ant"}, {nodeName: "b", nodeText: "ant"} ]
+        ]);
+
+        fluid.tests.setPageSize(pager, sequence, 4);
+        fluid.tests.verifyToolTips(pager, sequence, 4, [
+            [ {nodeName: "b", nodeText: "dog"}, {nodeName: "b", nodeText: "fish"} ],
+            [ {nodeName: "b", nodeText: "camel"}, {nodeName: "b", nodeText: "ant"} ]
+        ]);
```

Comment:
In looking at these now, I think you actually want to direct the tests with a structure that looks like:

```javascript
fluid.tests.tooltipTestDriver  = [
    {
        pageSize: 2,
        expectedTooltips: [
            {nodeName: "b", nodeText: "dog"}, {nodeName: "b", nodeText: "cat"}
        ], [ 
            {nodeName: "b", nodeText: "bird"}, {nodeName: "b", nodeText: "fish"}
        ], [
            {nodeName: "b", nodeText: "camel"}, {nodeName: "b", nodeText: "dragon"}
        ], [
            {nodeName: "b", nodeText: "ant"}, {nodeName: "b", nodeText: "ant"}
        ]
    },
    // and so on
];
```
This structure should be in the public namespace for potential reuse. You'd pass this structure into a function that loops over the top level array, and constructs and returns the sequence.